.eslintignore

```diff
@@ -3,3 +3,4 @@ coverage
 build
 lib
 **/aggregated-translations/*.*
+packages/terra-dev-site/src/terra-application-temporary
```

Comment:
Terra-dev-site v7 uses some components of terra-application that have not yet been released. These have been collected into the terra-application-temporary folder. This folder isn't linted and will be removed as we add features to terra-application v2